src/Host.cpp

```diff
@@ -932,20 +932,20 @@ std::tuple<bool, QString, QString> Host::saveProfile(const QString& saveFolder,
     emit profileSaveStarted();
     qApp->processEvents();
 
-    auto watcher = new QFutureWatcher<void>;
+    QFutureWatcher<void> watcher;
```

Comment:
How does this work then? Won't the `watcher` be gone when the method finishes, and thus the future will never get called?